lib/beaker-hostgenerator/data.rb

```diff
@@ -1455,6 +1455,16 @@ def osinfo
             'template' => 'win-2012r2-x86_64'
           }
         },
+        'windows2012r2fips-64' => {
```

Comment:
Do you need a ```windows2012r2fips-6432``` definition here as well?
Most of the other platforms include this as well.

Reply:
Added windows2012r2fips-6432 with its fixtures. Wasn't sure, but it seems it should be added.

Reply:
👍